### PR TITLE
docs(tabs): decompose manual selection section

### LINKED ISSUE #3545: docs(tabs): decompose Manual selection
Decompose Manual selection to Manual and Disable selection (according to #3267)

### PR DESCRIPTION
**Pull request by artolshansky**
Decompose Manual selection to Manual and Disable selection

close #3545

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

**Review by EvilAlexei — CHANGES_REQUESTED** [review]
PR checklist should be completed

File: demo/src/app/components/+tabs/tabs-section.list.ts [hunk 1]
```diff
@@ -45,6 +46,13 @@ export const demoComponentContent: ContentSection[] = [
         html: require('!!raw-loader?lang=markup!./demos/manual/manual.html'),
         outlet: DemoTabsManualComponent
       },
+      {
+        title: 'Disable selection',
+        anchor: 'tabs-disable',
```
Comment (EvilAlexei) [inline demo/src/app/components/+tabs/tabs-section.list.ts]: `tabs disable` and `disable selection` not the same, try not to change meaning of anchor.

**Review by EvilAlexei — CHANGES_REQUESTED** [review]

File: demo/src/app/components/+tabs/tabs-section.list.ts (hunk 1 above)
Comment (artolshansky) [reply demo/src/app/components/+tabs/tabs-section.list.ts]: fixed it

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Review by valorkin — DISMISSED** [review]

**Pull request merged**